provider/provider_test.go

```diff
@@ -45,7 +45,7 @@ func init() {
 
 	opendistroOriginalConfigureFunc := testAccOpendistroProvider.ConfigureContextFunc
 	testAccOpendistroProvider.ConfigureContextFunc = func(c context.Context, d *schema.ResourceData) (interface{}, diag.Diagnostics) {
-		err := d.Set("url", "http://admin:admin@127.0.0.1:9200")
+		err := d.Set("url", "http://admin:myStrongPassword123!@127.0.0.1:9200")
```

Comment:
Not sure if I should modify this/not sure how it's used - maybe in the integration test with opensearch 2.x?

Reply:
we can fetch this value from os if the tests are run in the docker container spun-up by the test workflow:
os.Getenv("OPENSEARCH_INITIAL_ADMIN_PASSWORD")

@prudhvigodithi correct me if I'm wrong here